manifestservice/manifests/__init__.py

```diff
@@ -305,9 +439,7 @@ def is_valid_manifest(manifest_json, required_keys):
     return True
 
 
-def _generate_unique_manifest_filename(
-    folder_name, manifest_bucket_name, users_existing_manifest_files
-):
+def _generate_unique_manifest_filename(users_existing_manifest_files):
```

Comment:
I'm finding this confusing. I understand that in the master branch that the first two arguments that got passed into this function is not being used, so removing them makes sense. But then you should also change the signature at where this function got called at https://github.com/uc-cdis/manifestservice/pull/43/files#diff-fabdcc5ef3a4225800a012be5195c800e5d318dd779a5694b68dcaa230632fb1R360